src/test/java/ch/liquidmind/inflection/grammar/IdentifierTest.java

```diff
@@ -0,0 +1,67 @@
+package ch.liquidmind.inflection.grammar;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import ch.liquidmind.inflection.compiler.util.InflectionCompilerTestUtility;
+import ch.liquidmind.inflection.test.AbstractInflectionTest;
+
+@RunWith( Parameterized.class )
+public class IdentifierTest extends AbstractInflectionTest
+{
+
+	@Parameters( name = "{index}: Identifier: {0}, expected compilation: {1}" )
+	public static Collection< Object[] > data()
+	{
+		return Arrays.asList( new Object[][] { { "A", true },
+
+				{ "_A", true },
```

Comment:
Either:
a) Comment why we are only testing a subset of first char / following char cases, or...
b) Systematically test all cases.